src/tests/TestSettingsJSONSerialization/generate_test_case.py

```diff
@@ -139,7 +139,7 @@ def getDefaultValueForType(dataType):
     elif dataType in ["IdleAction"]:
         return "Settings::Deafen"
     elif dataType in ["NoiseCancel"]:
-        return "Settings::NoiseCancelBoth"
+        return "Settings::NoiseCancelOff"
```

Comment:
I would move this into a separate commit.

Reply:
I believe that this is needed in order for tests to pass. If it was put into a separate commit, we'd have a commit for which tests can't pass.

Reply:
@guillermofbriceno If that's the case, let's keep the change in the same commit.